test/functional/setup/setup_dbs.py

```diff
@@ -75,5 +83,8 @@ def executeBulkCopy(conn_options, dbname, tblname, datafile):
     setupTestDatabase(conn_options, args.DBNAME, args.AZURE)    
     # populate these tables
     populateTables(conn_options, args.DBNAME)
+    # setup AE (certificate, column master key and column encryption key)
+    setupAE(conn_options, args.DBNAME)
+    
```

Comment:
I believe we should check if setting up for Azure here? If so, should we skip this step even running in Windows?